hydro_lang/src/live_collections/optional.rs

```diff
@@ -730,14 +730,6 @@ where
     pub fn then<U>(self, value: Singleton<U, L, Bounded>) -> Optional<U, L, Bounded> {
         value.filter_if_some(self)
     }
-
-    pub fn into_stream(self) -> Stream<T, L, Bounded, TotalOrder, ExactlyOnce> {
-        if L::is_top_level() {
-            panic!("Converting an optional to a stream is not yet supported at the top level");
-        }
-
-        Stream::new(self.location, self.ir_node.into_inner())
-    }
```

Comment:
Breaking change?

Reply:
Just moved to a separate `impl` group with effectively equivalent bounds (for symmetry with `Singleton`).